notebooks/en/enterprise_cookbook_gradio.ipynb

```diff
@@ -0,0 +1,5796 @@
+{
```

Comment:
<ul><li>You could give a link to the cookbook recipe here</li><li>also it should be <code>InferenceClient</code> repetitively</li><li>maybe give a link to where people can get their tokens and encourage them to create a fine-grained one for utmost security</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>